hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -816,65 +821,30 @@ protected void regionMoved(int region, int oldServer, int newServer) {
      *
      * @param stats the costs
      * @return a scaled set of costs.
-     */
-    protected double costFromArray(double[] stats) {
+     */ protected double costFromArray(double[] stats) {
```

Comment:
Something happened here? Did you mean this change?

Reply:
It was a deletion of eol. but I will put it back for styling